### ISSUE TITLE
How to control streamdeck from terminal

### ISSUE THREAD

**Issue by NeoCortex97** [timothycrosley/streamdeck-ui#299]
# Question

I would like to implement a view features that I could not think of how to implement by a script.
1. I have some menus and want to add a home button that returns me to the previewous page.
2. I would like to set the brightness based on Time.
3. I would like to change Pages based on currently focussed application.
    (This is for sake of this question a duplicate of 1.)
4. would like to automatically and transparently switch the "Profile" (a set of pages) based on workload.
    (Preferably without restarting streamdeck ui.)

From the docs I'm beginning to fear that it would be impossible.

**Comment by dodgyrabbit** [comment]
You are correct, these features are not currently possible. Thanks for the suggestions.

**Comment by dhtseany** [comment]
I too would find this feature quite useful as I do most of my live production from .sh scripts controlling the other apps on my system and it's be super cool to have reactions to button presses on the Streamdeck. 

As suggested by the OP being able to perform basic tile manipulation from the terminal would be a fantastic addition!

**Comment by 4s3ti** [comment]
hi, this project seems to be stale and a new fork as been made with the intention of replacing this one.

if this still an issue please reopen it at: https://github.com/streamdeck-linux-gui/streamdeck-linux-gui
